property/index.js

```diff
@@ -166,3 +161,62 @@ module.exports = yeoman.generators.Base.extend({
 
   saveProject: actions.saveProject
 });
+
+function coerceDefault(def, value) {
```

Comment:
As far as my understanding goes, the `def` object is a property definition in LoopBack format. I would like to see a more descriptive argument name here, as `def` can stand for both `default` and `definition`.